src/test/java/com/javagirls/MangoRoom/service/RoomServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package com.javagirls.MangoRoom.service;
+
+import com.javagirls.MangoRoom.dto.RoomDTO;
+import com.javagirls.MangoRoom.entity.Room;
+import com.javagirls.MangoRoom.repository.RoomRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+class RoomServiceTest {
+
+
+    @MockBean
+    private RoomRepository repository;
+    @Autowired
```

Comment:
proponuję użyć @InjectMock zamiast @Autowired

Reply:
Jak zmieniłam to na @InjectMocks to zwraca mi change room NullPointera

Reply:
dziwne, ale ok